lib/open_telemetry/tracer.ex

```diff
@@ -1,48 +1,91 @@
 defmodule OpenTelemetry.Tracer do
-  @moduledoc false
+  @moduledoc """
+  Defines tracer module.
 
+  ## Example
+
+      defmodule Tracer do
+        use #{inspect __MODULE__}
+      end
+  """
+
+  # TODO:
+  # - what options are possible there?
+  # - should we use `:otp_app` attribute to allow configuring tracer within
```

Comment:
Not sure what this means exactly, but the tracer should definitely be configured within an otp app. A tracer needs to have a name and that will be per-otp app

Reply:
Like in Ecto:

```elixir
defmodule Tracer do
  use OpenTelemetry.Tracer, otp_app: :my_app
end
```

And then in configuration:

```elixir
config :my_app, Tracer,
  handler: :ot_tracer_default
```